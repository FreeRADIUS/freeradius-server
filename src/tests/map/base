update request {
	Filter-Id := "filter"
	User-Name := "blah"

	&reply:Filter-Id += &request:Filter-Id[*]
}
