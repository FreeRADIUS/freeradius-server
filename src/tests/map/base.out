update request {
	&Filter-Id := "filter"
	&User-Name := "blah"
	&reply:Filter-Id += &Filter-Id[*]
}
